src/main/java/PersonalBudget/business/user/domain/controller/SignUpController.java

```diff
@@ -20,10 +21,15 @@
 public class SignUpController {
 
     private final UserService userService;
+    private final UserTemplateService userTemplateService;
+    public static final String SIGNUP = "signup/signup";
+    public static final String SUCCESS = "signup/success";
+    public static final String REDIRECT = "redirect:signup/success";
```

Comment:
the redirect should be build up from success variable. 
```suggestion
    public static final String REDIRECT = "redirect:" + SUCCESS
```
this way, if you ever change success we will not need to remember to change redirect as well. Please apply this comment to all redirects.